docs/index.adoc

```diff
@@ -250,54 +250,31 @@ To get the most out of `Cache`:
 NOTE: <<task,Task>> apps don't get a `Cache` parameter
 as they aren't meant to share the data between invokes.
 
-=== Store and load
+=== Get, set and delete
 
 [source,python]
 ----
 include::src/cache/tutorial001.py[]
 ----
-<.> `Cache` can store `str` values.
+<.> Save some data to `Cache`.
 <.> Load the value using its key.
-<.> `Cache` can store `int` values.
-<.> `Cache` stores the values in binary format.
-The data is decoded while loading,
-but `Cache` can't assume the type of the value.
-That's why the returned type is `str`.
-<.> `Cache` can store `float` values.
-<.> `Cache` can store `bytes` values.
-<.> Store multiple values at once using `mapping` parameter.
-<.> Load all data from cache.
```

Comment:
Added more examples, check them out.

Reply:
It will be printed only once for each location (module + line number). So should not be too excessive

https://docs.python.org/3.8/library/warnings.html#:~:text=Disposition-,%22default%22,-print%20the%20first

```python

import warnings


def func():
    warnings.warn("Hello", FutureWarning)


func()
func()

# > FutureWarning: Hello
# >  warnings.warn("Hello", FutureWarning)

```